crates/astria-sequencer/src/mempool/benchmarks.rs

```diff
@@ -285,8 +285,11 @@ fn run_maintenance<T: MempoolSize>(bencher: divan::Bencher) {
         .unwrap();
     // Set the new nonce so that the entire `REMOVAL_CACHE_SIZE` entries in the
     // `comet_bft_removal_cache` are filled (assuming this test case has enough txs).
-    // allow: this is test-only code, using small values, and where the result is not critical.
-    #[allow(clippy::arithmetic_side_effects, clippy::cast_possible_truncation)]
+    #[expect(
+        clippy::arithmetic_side_effects,
+        clippy::cast_possible_truncation,
+        reason = "this is test-only code, using small values, and where the result is not critical"
+    )]
```

Comment:
OK, this expect is so wordy, I think it'd be easier to just change all of these operations to use checked arithmetic and `unwrap`...